karaf/features/src/main/feature/feature.xml

```diff
@@ -315,16 +309,55 @@
         <!-- (3) -->
         <bundle dependency='true'>wrap:mvn:com.squareup.okhttp3/okhttp/3.12.6$Bundle-SymbolicName=squareup-okhttp3&amp;Bundle-Version=3.12.6&amp;Import-Package=okio;version=1.15,*;resolution:=optional</bundle>
         <bundle dependency='true'>wrap:mvn:com.squareup.okhttp3/logging-interceptor/3.12.6$Bundle-SymbolicName=squareup-okhttp3-logging-interceptor&amp;Bundle-Version=3.12.6&amp;Import-Package=*;resolution:=mandatory</bundle>
+
         <!-- (4) -->
         <!-- these are the main required bundles for these features above -->
         <bundle>mvn:io.fabric8/zjsonpatch/0.3.0</bundle>
         <bundle>mvn:io.fabric8/kubernetes-model/${kubernetes-client.version}</bundle>
         <bundle>mvn:io.fabric8/kubernetes-client/${kubernetes-client.version}/jar/bundle</bundle>
         <bundle>mvn:io.fabric8/openshift-client/${kubernetes-client.version}/jar/bundle</bundle>
 
+        <bundle dependency='true'>wrap:mvn:org.microbean/microbean-helm/2.12.4-SNAPSHOT/jar/$Bundle-SymbolicName=microbean-helm&amp;Bundle-Version=2.12.4-SNAPSHOT&amp;Import-Package=*;resolution:=mandatory</bundle>
```

Comment:
comment that all these are needed for microbean

Reply:
Added.